src/lib/components/Creatibutors/CreatibutorsField.js

```diff
@@ -0,0 +1,146 @@
+            icon={labelIcon}
+            label={label}
+          ></FieldLabel>
+          <List>
+            {getIn(values, fieldPath, []).map((value, index, array) => {
+              const key = `${fieldPath}.${index}`;
+              const personOrOrgPath = 'person_or_org';
+              const typeFieldPath = `${personOrOrgPath}.type`;
+              const familyNameFieldPath = `${personOrOrgPath}.family_name`;
+              const givenNameFieldPath = `${personOrOrgPath}.given_name`;
+              const nameFieldPath = `${personOrOrgPath}.name`;
+              const affiliationsFieldPath = 'affiliations';
+              // Default to person type
+              const isPerson =
+                _get(value, typeFieldPath, CREATIBUTOR_TYPE.PERSON) ===
+                CREATIBUTOR_TYPE.PERSON;
+              const displayName = isPerson
+                ? displayPersonName(
+                    _get(value, familyNameFieldPath, 'No family name'),
+                    _get(value, givenNameFieldPath, 'No given name'),
+                    _get(
+                      value,
+                      `${affiliationsFieldPath}[0].name`,
+                      'No affiliation name'
+                    )
+                  )
+                : _get(value, nameFieldPath, 'No organization name');
+              return (
+                <CreatibutorsFieldItem
+                  key={key}
+                  {...{
+                    displayName,
+                    index,
+                    roleOptions,
+                    schema,
+                    compKey: key,
+                    initialCreatibutor: value,
+                    removeCreatibutor: formikArrayRemove,
+                    replaceCreatibutor: formikArrayReplace,
+                    moveCreatibutor: formikArrayMove,
+                    addLabel: this.props.modal.addLabel,
+                    editLabel: this.props.modal.editLabel,
+                  }}
+                />
+              );
+            })}
+            <CreatibutorsModal
+              onCreatibutorChange={(selectedCreatibutor) => {
+                formikArrayPush(selectedCreatibutor);
+              }}
+              action="add"
+              addLabel={this.props.modal.addLabel}
+              editLabel={this.props.modal.editLabel}
+              roleOptions={roleOptions}
+              schema={schema}
+              trigger={
+                <Button type="button" basic secondary>
+                  <Icon name="add" />
+                  {this.props.addButtonLabel}
+                </Button>
+              }
+            />
+          </List>
+        </Form.Field>
+      </DndProvider>
+    );
+  }
+}
+
+export class CreatibutorsField extends Component {
+  render() {
+    return (
+      <FieldArray
+        name={this.props.fieldPath}
+        test={'test'}
+        component={(formikProps) => (
+          <CreatibutorsFieldForm {...formikProps} {...this.props} />
+        )}
```

Comment:
:+1: I think I prefer this technique for Formik wrapped field. It allows us to control the props passed down more effectively. It's error-prone and detrimental to understanding when the wrapped field is a method on the same class and therefore needs to pop some props to make sure they are not used in the `...uiProps` pattern we have been using.